<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.0" language="en_US">
<context>
    <name>SplashScreenClass</name>
    <message>
        <location filename="../SplashScreen.ui" line="69"/>
        <source>#terminal_not_available</source>
        <translation>The ATM is temporarily out of service</translation>
    </message>
    <message>
        <location filename="../SplashScreen.ui" line="134"/>
        <source>#updater_state_message</source>
        <translation>Maintenance work in progress</translation>
    </message>
</context>
<context>
    <name>UpdaterApp</name>
    <message>
        <location filename="../UpdaterApp.cpp" line="78"/>
        <source>#error_run_from_temp_dir</source>
        <translation>Failed to run updater from a temp path</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="79"/>
        <source>#error_connection_to_guard</source>
        <translation>Failed to connect to guard.exe</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="80"/>
        <source>#error_unknown_command</source>
        <translation>Unknown command</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="81"/>
        <source>#error_second_instance</source>
        <translation>Deny second updater instance</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="82"/>
        <source>#error_unknown</source>
        <translation>Unknown error</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="83"/>
        <source>#error_network</source>
        <translation>Network error</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="84"/>
        <source>#error_parse_response</source>
        <translation>Update server parse response error</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="85"/>
        <source>#error_deploy</source>
        <translation>Error while copying files</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="86"/>
        <source>#error_aborted</source>
        <translation>External abort</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="87"/>
        <source>#error_update_blocked</source>
        <translation>Update is blocked</translation>
    </message>
</context>
</TS>
